huaweicloud/services/acceptance/workspace/data_source_huaweicloud_workspace_tags_test.go

```diff
@@ -0,0 +1,175 @@
+package workspace
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccWorkspaceTagsDataSource_basic(t *testing.T) {
```

Comment:
```suggestion
func TestAccTagsDataSource_basic(t *testing.T) {
```
Do not include package name